### DISCUSSION TITLE
出码后如何在项目pinia的store中存取低代码里store变量？

### DISCUSSION THREAD

**Discussion by stwflyfox** [opentiny/tiny-engine#904] [Q&A]
![image](https://github.com/user-attachments/assets/d931df99-75d3-4c0e-b503-5282242fe39a)

![image](https://github.com/user-attachments/assets/ee6c66e5-75ed-40f1-b9b4-9b112558f217)

**Comment by chilingling** [comment]
出码后，globalState 全局的 pinia 与高代码的 `usexxx` store 差异不大，可以直接在高代码 `import { uesxxx } from 'path/to/lowcode/store'`。进行调用赋值。（可以手动出码看看看看）

或者引用  lowcode 文件夹下面的 store  `import { useStores } from 'path/to/lowcode/store.js'`

**Comment by stwflyfox** [comment]
已解决
